connector/src/main/resources/org/jboss/as/connector/subsystems/datasources/LocalDescriptions.properties

```diff
@@ -153,7 +153,7 @@ datasources.data-source.reauth-plugin-class-name=The fully qualified class name
 datasources.data-source.reauth-plugin-properties.reauth-plugin-class-name=The fully qualified class name of the reauthentication plugin implementation
 datasources.data-source.reauth-plugin-properties=The properties for the reauthentication plugin
 
-datasources.data-source.flush-strategy=Specifies how the pool should be flush in case of an error. Valid values are: FailingConnectionOnly (default), IdleConnections and EntirePool
```

Comment:
well there is a validator on flush strategy, but not a setAllowedValues()
I can do that if you require this change to merge the commit